extensions/custom_pages/elements/CalendarEntryElement.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\calendar\extensions\custom_pages\elements;
+
+use humhub\libs\Html;
+use humhub\modules\calendar\models\CalendarEntry;
+use humhub\modules\custom_pages\modules\template\elements\BaseContentRecordElement;
+use Yii;
+
+/**
+ * Class to manage content record of the Calendar event
+ *
+ * @property-read CalendarEntry|null $record
+ */
+class CalendarEntryElement extends BaseContentRecordElement
+{
+    protected const RECORD_CLASS = CalendarEntry::class;
+
+    /**
+     * @inheritdoc
+     */
+    public function getLabel(): string
+    {
+        return Yii::t('CalendarModule.template', 'Calendar event');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function attributeLabels()
+    {
+        return [
+            'id' => Yii::t('CalendarModule.template', 'Select calendar event'),
+        ];
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function render($options = [])
+    {
+        $result = Html::encode($this->record->title);
+
+        if ($this->isEditMode($options)) {
```

Comment:
It's more related to the Custom Pages module in general. but why we need the EditMode here? 

Since we only show the inline EditMode for Container, this should be necessary?

Reply:
@luke- This code is almost same to other similar elements, e.g. [TextElement](https://github.com/humhub/custom-pages/blob/develop/modules/template/elements/TextElement.php#L101-L113).

The wrapper is used on edit mode when the Element is not inside a Container, when it is inside a Template/Layout:

![render-element](https://github.com/user-attachments/assets/c7ee4ea5-dfeb-414a-95c3-aa142312fa0b)

Is it wrong? Do we really remove the editable wrappers from all Elements and keep them only for Containers?
Or should I remove the edit wrapper only for the Calendar Elements?

Reply:
I've created a new Issue for this.